README.md

```diff
@@ -25,3 +25,8 @@ Future Projects :<br>
   - _Opamp design and analysis, Two and multiple stages in sky130 + ptm models(Lower nodes) + cadence virtuoso_
   - _Voltage reference circuit with sky130pdk_
   - _More to come...._
+
+
+For anything further, feel free to reach out! 🙂
+
+### Visitor Count: ![Visitor Count](https://profile-counter.glitch.me/D-curs-D/count.svg)
```

Comment:
Visitor Count not necessary, since it's not accurate. 
Can you remove this?

Reply:
Sure